frontend/src/components/notification/notification.js

```diff
@@ -1,24 +1,31 @@
 import React, { useEffect, useState } from "react";
+import { AiOutlineClose } from "react-icons/ai";
 import "./notification.css";
 
 function Notification({
   notificationType,
   notificationMessage,
   setShowNotification,
+  exitable,
+  link,
 }) {
   const [showAlert, setShowAlert] = useState(true);
   useEffect(() => {
-    setTimeout(function () {
-      setShowAlert(false);
+    if (!exitable) {
       setTimeout(function () {
-        setShowNotification(false);
-      }, 250);
-    }, 5000);
+        setShowAlert(false);
+        setTimeout(function () {
+          setShowNotification(false);
```

Comment:
Timeout inside timeout ?

Reply:
It is in order to get the effect of fade out once the notification apprears